lib/src/main/kotlin/com/swmansion/starknet/data/types/Execution.kt

```diff
@@ -97,7 +97,23 @@ data class ExecutionParams(
 )
 
 @JvmSynthetic
-internal fun callsToExecuteCalldata(calls: List<Call>): List<Felt> {
+private fun callsToExecuteCalldataCairo1(calls: List<Call>): List<Felt> {
```

Comment:
Could u please expose both of `callsToExecuteCalldataX` as a global function. We are using it on our end and now that we have 2 version of it, I would want to avoid copy pasting it into our codebase.

Reply:
Sure

Reply:
Do you actually need both of the subfunctions, or is exposing just `callsToExecuteCalldata` enough?

Reply:
Temporarily we need both - one of the contracts that we have still uses the old encoding

Reply:
But that function supports both versions:

```kotlin
@JvmSynthetic
 internal fun callsToExecuteCalldata(calls: List<Call>, cairoVersion: Felt = Felt.ZERO): List<Felt> {
     if (cairoVersion == Felt.ONE) {
         return callsToExecuteCalldataCairo1(calls)
     }

     return callsToExecuteCalldataCairo0(calls)
 }
```

Reply:
Oh, that's what u meant. In that case - yeah just exposing this method to the clients will be enough